### DISCUSSION TITLE
Starting DepotDownloader.exe w/o any parameters prompts for required ones

### DISCUSSION THREAD

**Discussion by FatalBulletHit** [SteamRE/DepotDownloader#217] [Q&A]
Starting `DepotDownloader.exe` w/o any parameters (aka directly from the explorer by double clicking it) should prompt for required parameters, e.g. app, depot and manifest ids as well as username and password.

Leaving manifest id empty would use the latest manifest, leaving username empty would use anonymous login.

**Comment by SlejmUr** [comment]
>  "Leaving manifest id empty uses latest manifest"

Yes, it automatically use the latest manifest.

> "leaving username empty uses anonymous login"

I think you can actually use it, but with anonymous login you only able to download free game. (Probably , not tested)

**Reply by FatalBulletHit** [reply]
I'm aware of the current parameter behavior, this is a suggestion for when no parameters whatsoever are defined or the application is started from the explorer aka by double clicking it.

It would then still prompt for a manifest id or username, in case the user wants to specify those, but pressing enter w/o any input would revert to default behavior.
